addons/project/models/project.py

```diff
@@ -1946,6 +2021,8 @@ def create(self, vals_list):
         default_stage = dict()
         recurrences = self.env['project.task.recurrence']
         for vals in vals_list:
+            if not vals.get('name') and vals.get('display_name'):
+                vals['name'] = vals['display_name']
```

Comment:
Is it not the role of the inverse method to set the name with the display_name? 🤔

Reply:
the inverse method is called after inserting in db, as the name is a mandatory field, it can't be null
if i remove this code, i will get ERROR: null value in column "name" violates not-null constraint
![image](https://user-images.githubusercontent.com/64937092/223094694-255236c2-2210-421c-b4ba-6389ab093b58.png)